### DISCUSSION TITLE
Is it Possible to Create Standalone Containers Outside of Stacks?

### DISCUSSION THREAD

**Discussion by x1ao4** [louislam/dockge#408] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Dockge is a manager for docker-compose.yaml files that focuses on stacks. Currently, when you create containers using Dockge, they are always placed within a stack, even if the stack contains just one container.

I might want to create individual containers for specific projects that don't need to be part of a stack. Can Dockge support this?

### 🐻 Dockge Version

1.4.1

### 💻 Operating System and Arch

macOS 13.3.1

### 🌐 Browser

Google Chrome 120.0.6099.234

**Comment by louislam** [comment, marked as answer]
Please read FAQ (https://github.com/louislam/dockge#faq)

**Reply by x1ao4** [reply]
"If you still need to manage something like docker networks, single containers, the answer may be no."

I see, whether it's containers grouped in stacks or single containers, they are all managed using compose.yaml files. I'm not sure why support for single containers isn't available. It would be much better if it were supported.

However, I appreciate your efforts. Dockge is incredibly beginner-friendly, making it easy to operate and use. Thank you.
